cli/cmd/lql_create.go

```diff
@@ -0,0 +1,77 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+const (
```

Comment:
nitpick: Some of these output messages start with an upper case letter and some with lower case. Would be nice to have them consistent.

Reply:
?  They should all start with `lql`...

Reply:
Chiming in here. A bigger question is: Do we need do define constants for debug statements? 🤔 My gut tells me not but I am curious to know why you defined them as constants.

Reply:
These are strings that either print, or log.  They are constant and IMO easier to modify when declared in this matter.  Furthermore most of these sub commands follow a similar pattern for which it was easy to just declare sub command specific messages that drop into the code...

Reply:
@afiune and I discussed the merits of this approach and we're sorta on the same page that there are pro's and con's to doing it like this vs. not. He was reasonable enough to allow this approach to be pursued for now and potentially changed in the future.